backend/src/db/schemas/webhooks.ts

```diff
@@ -21,7 +21,11 @@ export const WebhooksSchema = z.object({
   keyEncoding: z.string().nullable().optional(),
   createdAt: z.date(),
   updatedAt: z.date(),
-  envId: z.string().uuid()
+  envId: z.string().uuid(),
+  encryptedUrl: z.string().nullable().optional(),
```

Comment:
May be keep the naming convention same by as `urlCipherText` or changing test as `encrypted` prepended